triples-generator/src/main/scala/io/renku/triplesgenerator/events/consumers/syncrepometadata/processor/model.scala

```diff
@@ -23,6 +23,8 @@ import io.renku.graph.model.images.{Image, ImageUri}
 import io.renku.graph.model.projects
 import io.renku.triplesstore.SparqlQuery
 
+import java.time.Instant
+
 private sealed trait DataExtract {
   val path:              projects.Path
   val name:              projects.Name
```

Comment:
I'm adding a note here but actually, I wanted to put it against the `maybeDateModified` val which is one line below. Should we maybe unwrap it, too, I mean take the Option out of it?

Reply:
Yes, sure if that is possible? I wasn't sure what is considered "required" in the response from GL.

Reply:
Ah, yes, you are right. I think we talked about that.